package.json

```diff
@@ -71,7 +71,7 @@
     "typescript": "4.6.4"
   },
   "peerDependencies": {
-    "puppeteer": "^15.1.1"
+    "puppeteer": "15.3.0"
```

Comment:
we should not pin the peer dependency: let's keep the `^` so that future semver compatible versions could also be used